ui/nuxeo-data-table/data-table-checkbox.js

```diff
@@ -27,13 +27,9 @@ import '../widgets/nuxeo-checkmark.js';
           :host(:focus) {
             outline: none;
           }
-
-          :host([header]) {
```

Comment:
I am not sure if I can simply remove this one, but it makes sense for me.

Reply:
This would be a breaking change if someone forked the data-table but did not fork the data-table-checkox, right? Should we consider instead to deprecate it?

Reply:
Yes, I think the safest is to deprecate this one. This was a weird behavior right from the start.